### ISSUE TITLE
A suggestion for a backwards and tilted ear type

### ISSUE THREAD

**Issue by Appw0** [exaskye/Ears#57]
Sort of like 'Above' but rotated backwards. Here is an example of what I have in mind, for a front, center, and back position.

![image](https://user-images.githubusercontent.com/18514637/147293395-692e3d39-d1bd-460f-af69-c9ffcd1a2c9e.png)
![image](https://user-images.githubusercontent.com/18514637/147293428-8de4ac1a-5260-4714-a674-902b089df54f.png)
![image](https://user-images.githubusercontent.com/18514637/147293437-297c0ee1-645f-4fef-a266-2a808b82142c.png)

A slider for the rotation would make this even better.

**Comment by exaskye** [comment]
Kind of overlaps with #50. If this were an ear mode, though, it could be 8x16 and have unique front/back faces.

**Comment by Appw0** [comment]
Yeah I saw #50, it kinda does. But I suppose at least the different positions are new.
